python/instrumentation/openinference-instrumentation-beeai/pyproject.toml

```diff
@@ -33,10 +33,10 @@ dependencies = [
 
 [project.optional-dependencies]
 instruments = [
-  "beeai-framework[duckduckgo,wikipedia]==0.1.18",
+  "beeai-framework[duckduckgo,wikipedia]==0.1.19",
```

Comment:
this is also very unusual in openinference. usually it is a min range, where it is exact here

Reply:
I agree - though it might be that the framework is a bit unstable. Feel free to just change to a min version. cc GALLLASMILAN

Reply:
thanks I set the min version to match what is in instruments `_instruments = ("beeai-framework >= 0.1.10",)` then made tests use min latest (0.1.19).

This will prevent disappearing spans which so far haven't happened with same caveat as all the other <1.x libraries here... there can be shifts and we want to keep tests running and respond if something breaks!